protocol/jsonRPC.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/dedis/student_21_pop/master/protocol/jsonRPC.json",
+    "title": "Match a custom JsonRpc 2.0 message",
+    "description": "Match a client query or a positive or negative server answer.",
+    "type": "object",
+    "properties": {
+        "jsonrpc": {
+            "type": "string",
+            "const": "2.0",
+            "description": "JSON RPC version"
+        },
+
+        "id": {
+            "$comment": "defined by the children"
+        }
+    },
+    "oneOf": [
+        {
+            "$ref": "answer/answer.json"
+        },
+        {
+            "$ref": "query/query.json"
+        }
+    ],
+    "required": ["jsonrpc", "id"]
```

Comment:
`id` is not required for a JSON-RPC notification. It's wrong to add it at this level.